bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -28,7 +29,6 @@ private func dummy() {
     _ = NSLocalizedString("ADD IMAGE", comment: "")
     _ = NSLocalizedString("Add image or video", comment: "")
     _ = NSLocalizedString("ADD IMAGE OR VIDEO", comment: "")
-    _ = NSLocalizedString("Add link", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/e393d55cacc030f43b7f64104a8ff1aa2b179a62/packages/block-library/src/navigation-link/edit.js#L616)

This string is already translated as it's used in the web version, so we don't need to include it here.